x/gov/client/cli/util.go

```diff
@@ -131,7 +131,7 @@ func AddGovPropFlagsToCmd(cmd *cobra.Command) {
 	cmd.Flags().String(FlagMetadata, "", "The metadata to include with the governance proposal")
 	cmd.Flags().String(FlagTitle, "", "The title to put on the governance proposal")
 	cmd.Flags().String(FlagSummary, "", "The summary to include with the governance proposal")
-	cmd.Flags().Bool(FlagExpedited, false, "Whether to expedite the governance proposal")
+	// cmd.Flags().Bool(FlagExpedited, false, "Whether to expedite the governance proposal") // cannot be enabled because of IBC redefining this flag in `upgrade-channels` command.
```

Comment:
Unrelated but should be backported to 0.53 and 0.50.
This should be uncommitted once this is gone, so that everyone can enjoy that flag being set.